<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IMainWindow</class>
 <widget class="QMainWindow" name="IMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qmapshack/resources.qrc">
    <normaloff>:/icons/48x48/QMapTool.png</normaloff>:/icons/48x48/QMapTool.png</iconset>
  </property>
  <widget class="CCanvas" name="canvas"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="actionSetupExtTools"/>
    <addaction name="actionSetupUnits"/>
    <addaction name="actionSetupCoordFormat"/>
    <addaction name="actionShowToolHelp"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFlipMouseWheel"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuSetup"/>
   <addaction name="menuView"/>
   <addaction name="menuWindow"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockTools">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="windowTitle">
    <string>Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="CToolStack" name="toolStack"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockShell">
   <property name="features">
    <set>QDockWidget::DockWidgetFeatureMask</set>
   </property>
   <property name="windowTitle">
    <string>Shell</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="CShell" name="textBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../qmapshack/resources.qrc">
     <normaloff>:/icons/32x32/Info.png</normaloff>:/icons/32x32/Info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSetupExtTools">
   <property name="icon">
    <iconset resource="../qmapshack/resources.qrc">
     <normaloff>:/icons/32x32/Apply.png</normaloff>:/icons/32x32/Apply.png</iconset>
   </property>
   <property name="text">
    <string>Ext. Tools</string>
   </property>
   <property name="toolTip">
    <string>Setup paths to external tools, like gdalwarp etc.</string>
   </property>
  </action>
  <action name="actionFlipMouseWheel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../qmapshack/resources.qrc">
     <normaloff>:/icons/32x32/MouseWheel.png</normaloff>:/icons/32x32/MouseWheel.png</iconset>
   </property>
   <property name="text">
    <string>Flip Mouse Wheel</string>
   </property>
   <property name="toolTip">
    <string>Flip Mouse Wheel</string>
   </property>
  </action>
  <action name="actionSetupUnits">
   <property name="icon">
    <iconset resource="../qmapshack/resources.qrc">
     <normaloff>:/icons/32x32/UnitSetup.png</normaloff>:/icons/32x32/UnitSetup.png</iconset>
   </property>
   <property name="text">
    <string>Setup Units</string>
   </property>
   <property name="toolTip">
    <string>Setup Units</string>
   </property>
  </action>
  <action name="actionSetupCoordFormat">
   <property name="icon">
    <iconset resource="../qmapshack/resources.qrc">
     <normaloff>:/icons/32x32/SetupCoordFormat.png</normaloff>:/icons/32x32/SetupCoordFormat.png</iconset>
   </property>
   <property name="text">
    <string>Setup Coord. Format</string>
   </property>
   <property name="toolTip">
    <string>Change the format coordinates are displayed</string>
   </property>
  </action>
  <action name="actionShowToolHelp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Tool Help</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="../qmapshack/resources.qrc">
     <normaloff>:/icons/32x32/Help.png</normaloff>:/icons/32x32/Help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CShell</class>
   <extends>QTextBrowser</extends>
   <header>shell/CShell.h</header>
  </customwidget>
  <customwidget>
   <class>CCanvas</class>
   <extends>QWidget</extends>
   <header>canvas/CCanvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CToolStack</class>
   <extends>QStackedWidget</extends>
   <header>tool/CToolStack.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qmapshack/resources.qrc"/>
 </resources>
 <connections/>
</ui>
